Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -241,7 +242,7 @@ void DecreaseVolume(unsigned short offset)
 
 void ToggleMuteVolume()
 {
-  bool& isMuted = SConfig::GetInstance().m_IsMuted;
+  bool isMuted = Config::Get(Config::MAIN_AUDIO_MUTED);
```

Comment:
This doesn't do the same thing anymore. (Note how the reference is assigned to in the next line.) Same with `currentVolume` in two places above.

Reply:
Ahh, I understand why a reference was used here now. Will fix this.